src/components/main-form.js

```diff
@@ -50,6 +50,35 @@ option2.term = 30
 
 // CSS
 
+const Summary = styled(Row)`
+  b {
+    font-weight: 500;
+  }
+
+  p,
+  h5 {
+    font-size: 17px;
+  }
+
+  h5 {
+    margin-bottom: 1rem;
+    margin-left: 1.5rem;
+  }
+
+  .col {
+    margin-left: -2rem;
+  }
```

Comment:
Ah, something I'm seeing is that you may not these custom. It's hidden in the design, but the explanation also has padding, which is why things align:

![image](https://user-images.githubusercontent.com/3756871/64914750-7c823c00-d70c-11e9-8a3e-43bfb1264fcf.png)

Reply:
not quite following your thought process here. Can you elaborate?

Reply:
Hm, since it's tricky to pair right now, here's how it may look in code:

https://github.com/jgozal/moneycompass/pull/31/files#diff-13bae51262f519caebe5af0bd2fcb33cR552

If the summary itself has padding, then we can follow standard row column padding for Bootstrap.

Please run the example locally if you want to see what I mean!